firmware/controllers/map_averaging.cpp

```diff
@@ -170,6 +178,18 @@ static void endAveraging(void *arg) {
 #if EFI_PROD_CODE || defined(__DOXYGEN__)
 	v_averagedMapValue = adcToVoltsDivided(
 			mapAccumulator / mapMeasurementsCounter);
+	if (mapMinBufferLength > 1) {
```

Comment:
I believev this line is not needed, now that 'mapMinBufferLength ' is always at least 1 I would prefer that the same code is executed even if mapMinBufferLength  == 1